apps/portals/standards/src/pages/StandardsDetailsPage.tsx

```diff
@@ -28,7 +42,58 @@ export const standardDetailsPageContent: DetailsPageContentType = [
     id: 'About The Standard',
     title: 'About The Standard',
     element: (
-      <MarkdownSynapseFromColumnData columnName={'studyDescriptionLocation'} />
+      <DetailsPageContextConsumer columnName={'Name'}>
+        {({ value, context }) => {
+          const query: Query = {
+            sql: dataSql,
+            additionalFilters: getAdditionalFilters(
+              undefined,
+              { Name: value! },
+              ColumnSingleValueFilterOperator.EQUAL,
+            ),
+          }
+          if (context.rowData && context.rowSet) {
+            return (
+              <RowDataTable
+                rowData={
+                  context.rowData.values?.filter(
+                    (value): value is string => value !== null,
+                  ) ?? []
```

Comment:
Would prefer that the filter is applied in the component (couldn't it also mess up the mapping of column -> data?). Remove the filter here and update the type in the component prop